### PR TITLE
Feat : Added home page

### PR DESCRIPTION
**Pull request by sushil-mod**
Added:
1.home page
2. footer

### CODE REVIEW

**Review by ozairkhan2810 — APPROVED** [review]
remove commented part in css file, rest looks good

File: style.css [hunk 1]
```diff
@@ -1,2 +1,188 @@
...
+  .padd-btm-lg{
+    padding-bottom: 1.5rem;
+  }
+  .padd-top-sm{
+    padding-top: 0.5rem;
+  }
+  
+  .padd-md{
+  padding: 1rem;
+  } */
```
Comment (rushikesh3299) [inline style.css]: If you dont want to use this in future you can remove it

**Review by rushikesh3299 — APPROVED** [review]
Looks Good to Me.

File: style.css (hunk 1 above)
Comment (sushil-mod) [reply style.css]: ooh i missed , thanks for pointing out

**Pull request merged**